projects/plugins/jetpack/_inc/client/newsletter/email-settings.jsx

```diff
@@ -52,26 +57,62 @@ const EmailSettings = props => {
 
 	const handleSubscriptionReplyToChange = useCallback(
 		value => {
-			updateFormStateAndSaveOptionValue( REPLY_TO_OPTION, value );
+			if ( value !== 'custom' ) {
+				updateFormStateAndSaveOptionValue( { [ REPLY_TO_OPTION ]: value, [ REPLY_TO_EMAIL ]: '' } );
+			} else {
+				updateFormStateAndSaveOptionValue( REPLY_TO_OPTION, value );
+			}
 		},
 		[ updateFormStateAndSaveOptionValue ]
 	);
 
+	const [ isEmailValid, setIsEmailValid ] = useState( 'not-checking' );
+	const isTypingEmailTimeoutRef = useRef( 0 );
+
+	const handleEmailBlur = useCallback(
+		event => {
+			if ( emailValidator.validate( event.target.value ) ) {
```

Comment:
Not a biggie, and both methods are fine by me: alternatively you could hook into input's own native browser validation since it has `type="email"` already, and even set the input "invalid" natively.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/checkValidity
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/reportValidity
https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/setCustomValidity

Or combine custom inline message but rely on checkValidity to validate the email.